fieldsolver/gridGlue.cpp

```diff
@@ -113,8 +115,8 @@ void filterMoments(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid,
                for (int c=-kernelOffset; c<=kernelOffset; c++){
                   for (int b=-kernelOffset; b<=kernelOffset; b++){
                      for (int a=-kernelOffset; a<=kernelOffset; a++){
-                        auto* cell {momentsGrid.get(i+a,j+b,k+c)};
                         for (int e = 0; e < fsgrids::moments::N_MOMENTS; ++e) {
+                           const auto* cell {momentsGrid.get(i+a,j+b,k+c)};
```

Comment:
I agree this can be const, but why put it inside the e loop?

Reply:
Yep, moved that back up in scope